widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -1607,6 +1607,11 @@ button#schedule-dialog-select-button {
   padding: 0px;
   display: none;
 }
+/* Uncomment the below style to validate whether frame is covered or uncovered */
```

Comment:
Do you think this is the right place for this comment/documentation?

Would a better place be in a wiki or QA validation steps or README.md ?

Reply:
Closing this PR, the above comment either has to be moved into xul.css(which would be shared by all themes) from Pentaho-Commons-XUL project or to be in wiki.  Will discuss on this topic in today's mid sprint whether to keep in wiki or   in the XUL-Project